src/test/java/com/artipie/RpPermissionsTest.java

```diff
@@ -101,4 +97,17 @@ void adminCanDoAnything() {
         );
     }
 
+    /**
+     * Permissions from repo-full-config.yml example file.
+     *
+     * @return Permissions parsed from file.
+     */
+    private YamlPermissions permissions() throws IOException {
+        return new YamlPermissions(
+            Yaml.createYamlInput(new File("src/test/resources/repo-full-config.yml"))
```

Comment:
@olegmoz let's use `TestResource` from asto to obtain content from config file

Reply:
@olenagerasimova thanks, corrected